Demo/RCHSearchViewController.swift

```diff
@@ -19,8 +19,8 @@ class RCHSearchViewController: UIViewController, UISearchBarDelegate, UICollecti
     @IBOutlet weak var searchProductsImageView: UIImageView!
     @IBOutlet weak var autocompleteTableView: UITableView!
     
-    var productArray: [RCHRecommendedProduct] = []
-    var autocompleteArray: [String] = ["Boots", "Bomber", "Boucle", "Boyfriend"]
+    var productArray: [RCHSearchProduct] = []
+    var autocompleteArray: [String] = []
```

Comment:
Don't include the type in the variable name. I'd use something like `products` and `autocompleteSuggestions`